src/extractor/edge_based_graph_factory.cpp

```diff
@@ -220,13 +217,16 @@ void EdgeBasedGraphFactory::InsertEdgeBasedNode(const NodeID node_u, const NodeI
         BOOST_ASSERT(forward_data.edge_id != SPECIAL_NODEID ||
                      reverse_data.edge_id != SPECIAL_NODEID);
 
+        m_compressed_edge_container.AddUncompressedEdge(edge_id_1, node_v, forward_data.distance);
```

Comment:
This *is* after the graphcompressor run.  At this stage, the CompressedEdgeContainer is full of normal compressed geometries.  This loop is our opportunity to iterate over all edge-based-nodes and test whether they contain compressed geometries, or whether they didn't get touched by the GraphCompressor and need to be added to the CompressedEdgeContainer.